doc/source/programs/gdal_raster_proximity.rst

```diff
@@ -0,0 +1,90 @@
+.. _gdal_raster_proximity:
+
+================================================================================
+``gdal raster proximity``
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Produces a raster proximity map.
+
+.. Index:: gdal raster proximity
+
+Synopsis
+--------
+
+.. program-output:: gdal raster proximity --help-doc
+
+Description
+-----------
+
+:program:`gdal raster proximity` generates a raster proximity map indicating the distance from
+the center of each pixel to the center of the nearest pixel identified as a target pixel.
+Target pixels are those in the source raster for which the raster pixel value is in the set of
+target pixel values.
+
+Standard options
+++++++++++++++++
+
+.. include:: gdal_options/of_raster_create_copy.rst
+
+.. include:: gdal_options/co.rst
+.. include:: gdal_options/overwrite.rst
+
+.. include:: gdal_options/ot.rst
+
+.. option:: -b, --band <BAND>
+
+    Input band (1-based index)
+
+.. option:: --target-values <TARGET-VALUES>
+
+    A single value or a comma separated list of target pixel values in the source image
+    to be considered target pixels.
+    If not specified, all non-zero pixels will be considered target pixels.
+
+.. option:: --max-distance <MAX-DISTANCE>
+
+    Maximum distance. The nodata value will be used for pixels beyond this distance.
+    Distance is interpreted in pixels unless `--distance-units geo` is specified.
+
+.. option:: --distance-units <pixel|geo>
+
+    Units for the distance. If `geo` is specified, the distance will be calculated in
+    georeferenced units (meters or degrees) using the pixel size of the input raster.
+    Otherwise, the distance is interpreted in pixels.
+
+.. option:: --fixed-buffer <FIXED-BUFFER>
```

Comment:
This option name isn't very clear to me (although the description is.) Would `--burn` be appropriate, as in `gdal vector rasterize` ? Or `--write-value` ?

Reply:
I'm happy to change it, `burn` means nothing to me (besides reminding of a fire). Wouldn't perhaps `--fixed-value` be more appropriate to convene the meaning that we are writing a fixed value instead of the varying distance?

Reply:
`--fixed-value` sounds good.